framework.py

```diff
@@ -20,5 +20,33 @@ def init_framework(self):
     
     def destroy_framework(self):
         self.log_debug("%s: Destroying..." % self)
+
+    ##########################################################################################
+    # public methods
+
+    def get_hyperlink_html(self, url, name):
```

Comment:
Wondering if this is the right place for the method within the framework. Perhaps adding `util` module to the fw would be better? Or maybe that's just overkill for this one method. Curious what @manneohrstrom thinks.